.changeset/late-worms-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+"remix": patch
```

Comment:
This one's not needed I think?

```suggestion
```

Reply:
technically no its not needed, but we release everything in lock step currently 🤷

Reply:
Yeah I know, that's why I approved it

Just wanted to let you know, as I thought you might have missed it somehow 😉